cmd/cosign/cli/sign/sign.go

```diff
@@ -67,7 +68,36 @@ const TagReferenceMessage string = `Image reference %s uses a tag, not a digest,
     images by tag will be removed in a future release.
 `
 
-func ShouldUploadToTlog(ctx context.Context, ko options.KeyOpts, ref name.Reference, tlogUpload bool) bool {
+const (
+	// spacing is intentional to have this indented
+	privacyStatement = `
+        Note that there may be personally identifiable information associated with this signed artifact.
+        This may include the email address associated with the account with which you authenticate.
+        This information will be used for signing this artifact and will be stored in public transparency logs and cannot be removed later.`
+	privacyStatementConfirmation = "By typing 'y', you attest that you grant (or have permission to grant) and agree to have this information stored permanently in transparency logs."
```

Comment:
The extra space got deleted

Reply:
Yep, with the new UI package confirmation the space was creating an indent that didn't look right to me.

Reply:
SG, I haven't tried out cosign since that was added so if it looks good, great!